### PR TITLE
Fix ETH Supply Simulator marker logic

### PR DESCRIPTION
**Pull request by marthendalnunes**

### CODE REVIEW

**Comment by BenSchZA** [comment]
@marthendalnunes could you clear all outputs from the following notebook, to more easily confirm what changes were made there? `experiments/notebooks/3_network_issuance_and_inflation_rate.ipynb`

File: experiments/notebooks/visualizations/__init__.py [hunk 1]
```diff
@@ -698,11 +698,22 @@ def fig_add_stage_markers(df, column, fig, secondary_y=None, parameters=paramete
...
+            ]
+            x_datetime = [nearest_row["timestamp"][0]]
+            y_value = [nearest_row[column][0]]
+
         fig.add_trace(
             go.Scatter(
                 mode="markers+text",
-                x=[date],
-                y=[df.loc[date.strftime("%Y-%m-%d")][column][0]],
+                x=x_datetime,
```
Comment (BenSchZA) [inline experiments/notebooks/visualizations/__init__.py:715]: I suggest keeping the list syntax here to make the code more clean.

File: experiments/notebooks/visualizations/eth_supply_simulator.py [hunk 2]
```diff
@@ -115,7 +115,7 @@
                         75: '75',
                         100: '100'
                     },
-                    value=25,
+                    value=30,
```
Comment (BenSchZA) [inline experiments/notebooks/visualizations/eth_supply_simulator.py:118]: Can we set these defaults values at the top of the file, and pass in to both the variable and the annotations? Will make updates more seamless in future.

**Comment by BenSchZA** [comment]
Final point of review - you're re-executed the ETH supply/inflation chart and it looks good? :)

File: experiments/notebooks/visualizations/eth_supply_simulator.py (hunk 2 above)
Comment (marthendalnunes) [reply experiments/notebooks/visualizations/eth_supply_simulator.py:118]: Sure, we can add a variable for `default_basefee`, and the same for the other dropdowns

**Comment by marthendalnunes** [comment]
@BenSchZA it looks like that 
image

**Review by BenSchZA — APPROVED** [review]

**Pull request merged**